fdbcli/fdbcli.actor.cpp

```diff
@@ -639,11 +639,11 @@ void initHelp() {
 	                                 "namespace for all the profiling-related commands.",
 	                                 "Different types support different actions.  Run `profile` to get a list of "
 	                                 "types, and iteratively explore the help.\n");
-	helpMap["cache_range"] = CommandHelp(
-	    "cache_range <set|clear> <BEGINKEY> <ENDKEY>",
-	    "Mark a key range to add to or remove from storage caches.",
-	    "Use the storage caches to assist in balancing hot read shards. Set the appropriate ranges when experiencing "
-	    "heavy load, and clear them when they are no longer necessary.");
+	helpMap["throttle"] =
+	    CommandHelp("throttle <on|off|enable auto|disable auto|list> [ARGS]",
+	                "view and control throttled tags",
+	                "Use `on' and `off' to manually throttle or unthrottle tags. Use `enable auto' or `disable auto' "
+	                "to enable or disable automatic tag throttling. Use `list' to print the list of throttled tags.\n");
```

Comment:
Do I miss something? I'm not sure I understand why `cache_range` becomes `throttle` here instead of being deleted.

Reply:
Nice catch. Thanks
Somehow messed this up from cherry-pick. Fixed.